.github/workflows/oatconf-AutoDeployTrigger-bbe0a634-b5af-496e-a6cc-4ea70da42805.yml

```diff
@@ -57,4 +57,4 @@ jobs:
           inlineScript: |
             az config set extension.use_dynamic_install=yes_without_prompt
             CONTAINER_APP_IDS=$(az resource list --tag stage=dev --tag image=openastrotech/config-server --query "[*].id | join(' ', @)" | sed 's|"||g')
-            az containerapp update --ids "$CONTAINER_APP_IDS" --image openastrotech/config-server:${{ github.sha }}
+            echo $CONTAINER_APP_IDS | xargs az containerapp update --image openastrotech/config-server:${{ github.sha }} --ids

```

Comment:
Need to surround `$CONTAINER_APP_IDS` with quotes: `echo "$CONTAINER_APP_IDS"`
Also why change to xargs?

Reply:
Because azure cli seems to be very stupid. The param `--ids` takes multiple IDs but handles only the first one 🤷

Reply:
lmao